app/components/results-list/template.hbs

```diff
@@ -29,14 +29,15 @@
                 </div>
             </div>
         </div>
-        {{else}}
+    {{else}}
         <h3 style="text-align: center;">
             No results found.
         </h3>
     {{/each}}
-    <div style="text-align: center; padding: 30px;">
-        <button class="btn btn-primary" {{action "pageback"}}>{{fa-icon "chevron-left"}}</button>
-        <button class="btn btn-primary" {{action "pagenext"}}>{{fa-icon "chevron-right"}}</button>
-    </div>  
-
+    {{#if data}}
```

Comment:
Ember will treat an empty array as falsy but in other places you are checking for data.length.